language-server/src/main/scala/scala/meta/languageserver/Compiler.scala

```diff
@@ -0,0 +1,113 @@
+package scala.meta.languageserver
+
+import java.io.File
+import java.nio.file.Files
+import java.util.Properties
+import scala.collection.mutable
+import scala.reflect.io
+import scala.tools.nsc.Settings
+import scala.tools.nsc.interactive.Global
+import scala.tools.nsc.reporters.StoreReporter
+import com.typesafe.scalalogging.LazyLogging
+import langserver.core.Connection
+import langserver.messages.MessageType
+import monix.reactive.Observable
+import org.langmeta.io.AbsolutePath
+
+case class CompilerConfig(
```

Comment:
Don't you need the scala version here?

Reply:
We will need to classload the compiler for it to work for more scala versions, that's the next step. Currently it's hardcoded for 2.12.3. We will likely only support the 2.11/2.12 releases supported by semanticdb, in order to provide scalafix linting as you type alongside completions.